modules/op-release-notes-1-8.adoc

```diff
@@ -50,8 +190,47 @@ Workaround: Use `nodejs:14-ubi8` instead of the `nodejs:14-ubi8-minimal` image s
 
-// [id="fixed-issues-1-8_{context}"]
-// == Fixed issues
+[id="fixed-issues-1-8_{context}"]
+== Fixed issues
+
+* Previously, with the resource pruner, each namespace operator created a command that ran in a separate container. This design consumed too many resources in clusters with a high number of namespaces. For example, to run a single command, a cluster with 1000 namespaces produced 1000 containers in a pod.
++
+This update fixes the issue. It passes the namespace-based configuration to the job so that all the commands run in one container in a loop. (link:https://issues.redhat.com/browse/SRVKP-2160[SRVKP-2160])
+// Pradeep Kumar
+
+// .Pipelines
+
+* Before this update, the metrics for pipeline runs in the Developer view of the web console were incomplete and outdated. With this update, the issue has been fixed so that the metrics are correct.
+// link:https://issues.redhat.com/browse/SRVKP-1789
+//  Khurram Baig
+
+* Before this update, if a pipeline had two parallel tasks that failed and one of them had `retries=2`, the final tasks never ran, and the Pipeline failed from a timeout. For example, the `pipelines-operator-subscription` task failed intermittently with an "Unable to connect to the server: EOF" error. With this update, the issue has been fixed so that the final tasks always run.
+// link:https://issues.redhat.com/browse/SRVKP-1724
+// Vincent Demeester
+
+* Before this update, when a pipeline run was in a "stopping" state, that is, if a task run failed, other task runs might not complete their retries. As a result, no `finally` tasks were scheduled, which caused the pipeline to hang. This update resolves the issue. `TaskRuns` and `Run` objects can retry when a pipeline run has stopped, even by  graceful stopping, so that pipeline runs can complete.
+// link:https://github.com/tektoncd/pipeline/pull/4651
+// Lee Bernick @lbernick
+
+* This update changes how resource requirements are calculated when one or more `LimitRange` objects are present in the namespace where a `TaskRun` object is. With this update, the scheduler considers `step` containers and excludes all other app containers, such as sidecar containers, when factoring requests from `LimitRange` objects.
+// link:https://github.com/tektoncd/pipeline/issues/4996
+// Simon Kaegi @skaegi
 
-// Pipelines
-// *
+* Before this update, under specific conditions, the flag package might incorrectly parse a subcommand immediately following a double dash flag terminator, `--`. In that case, it ran the entrypoint subcommand instead of the actual command. This update fixes this flag-parsing issue so that the entrypoint runs the correct command.
+// link:https://github.com/tektoncd/pipeline/issues/5095
+// Vincent Demeester @vdeemester
```

Comment:
@vdeemester Please review and approve.

Reply:
Please ignore the fact that GitHub closed this PR and leave your review here anyway.